src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
```

Comment:
I could, but then I just have to convert it to a String immediately at all the callers, since they're interpolating other strings. I don't feel super strongly.